Firestore/core/src/core/query.cc

```diff
@@ -67,14 +67,16 @@ bool Query::MatchesAllDocuments() const {
            explicit_order_bys_.front().field().IsKeyFieldPath()));
 }
 
-const FieldPath* Query::InequalityFilterField() const {
+const std::set<model::FieldPath> Query::InequalityFilterFields() const {
+  std::set<FieldPath> result;
```

Comment:
std::set here sorted the fields by the FieldPath as expected. Do I need to use SortedSet?  @ehsannas

Reply:
yes please!~~~